src/GitHub.InlineReviews/Peek/InlineCommentPeekResultPresentation.cs

```diff
@@ -40,9 +40,9 @@ private set
             }
         }
 
-        public event EventHandler IsDirtyChanged;
-        public event EventHandler IsReadOnlyChanged;
-        public event EventHandler<RecreateContentEventArgs> RecreateContent;
+        public event EventHandler IsDirtyChanged = delegate { }; // IsDirty is always false.
```

Comment:
I think I'd prefer these unused events to be declared as explicit interface implementations with empty `add` and `remove` blocks, even though it's a bit more verbose - this syntax doesn't make clear to me what is happening...

Reply:
Indeed. It's concise, but not exactly clear!